src/index.js

```diff
@@ -2,0 +1,43 @@
+// 'use strict';
+
+// import express from 'express';
+// import cors from 'cors';
+
+// const app = express();
+
+// app.use(cors({ origin: '*' }));
+// app.use(express.json());
```

Comment:
The middleware for parsing JSON requests is commented out. Uncomment this line to handle JSON payloads in requests.